sdcm/utils/remote_logger.py

```diff
@@ -299,6 +300,39 @@ def _logger_cmd(self) -> str:
         return f"{cmd} >> {self._target_log_file} 2>&1"
 
 
+class KubernetesWrongSchedulingLogger(CommandClusterLoggerBase):
+    restart_delay = 120
+    WRONG_SCHEDULED_PODS_MESSAGE = "Not allowed pods are scheduled on Scylla node found"
+
+    @property
+    def _logger_cmd(self) -> str:
+        if not self._cluster.allowed_labels_on_scylla_node:
+            return ''
+
+        wrong_scheduled_pods_on_scylla_node = []
+
+        node_names = [node.metadata.name for node in self._cluster.pools[self._cluster.SCYLLA_POOL_NAME].nodes.items]
+
+        try:
+            for pod in KubernetesOps.list_pods(self._cluster,
+                                               field_selector=f"spec.nodeName in ({', '.join(node_names)})"):
+                pod_allowed = False
+                for key, value in self._cluster.allowed_labels_on_scylla_node:
+                    if (key, value) in pod.metadata.labels.items():
+                        pod_allowed = True
+                if not pod_allowed:
+                    wrong_scheduled_pods_on_scylla_node.append(f"{pod.metadata.name} ({pod.nodeName} node)")
+        except Exception as details:  # pylint: disable=broad-except
+            self._log.debug("Failed to get pods list: %s", str(details))
+
+        if wrong_scheduled_pods_on_scylla_node:
```

Comment:
it's better to do the opposite: check for the list emptiness and return the empty string.

Reply:
done